src/olympia/users/models.py

```diff
@@ -119,7 +116,7 @@ class UserProfile(OnChangeMixin, ModelBase, AbstractBaseUser):
     email = models.EmailField(unique=True, null=True, max_length=75)
 
     averagerating = models.CharField(max_length=255, blank=True, null=True)
-    bio = NoLinksField(short=False)
+    biography = models.TextField(blank=True, null=True)
```

Comment:
We should document, here and in the API docs, that this field can contain some HTML (and, in the case of the API docs, mention that it should be the client responsibility to clean it before display)